config/dev/kustomization.yaml

```diff
@@ -1,14 +1,24 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: capi-migration-system-dev
-namePrefix: capi-migration-
+namespace: giantswarm
+namePrefix: capi-migration-dev-
+nameSuffix: -kopiczko
```

Comment:
Should this end up to final version?

Reply:
I can change that to anything else it will be overwritten by https://github.com/giantswarm/capi-migration/pull/23/files#diff-71caa12c7addf8197e109e323f5af04078e54a80afc210a71c18317f0d587dd3R33 which defaults to $USER (as described in README.md) once you run `make deploy`.